tests/tests_geomstats/test_riemannian_mean_shift.py

```diff
@@ -106,3 +106,40 @@ def test_double_cluster_riemannian_mean_shift(self):
         self.assertEqual(
             combined_cluster.shape[0], count_in_first_cluster + count_in_second_cluster
         )
+
+    @geomstats.tests.np_and_autograd_only
+    def test_predict_labels_riemannian_mean_shift(self):
```

Comment:
Up until `rms.fit` the code is the same as for `test_double_cluster_riemannian_mean_shift`. You can create a private function to avoid code duplication (a similar example [here](https://github.com/geomstats/geomstats/blob/master/tests/tests_geomstats/test_riemannian_kmeans.py)).

Reply:
Addressed this in the new PR